tests/IVIMmodels/unit_tests/test_ivim_fit.py

```diff
@@ -114,24 +114,31 @@
 
 def signal_helper(signal):
     signal = np.asarray(signal)
-    signal = np.abs(signal)
-    signal /= signal[0]
-    ratio = 1 / signal[0]
-    return signal, ratio
+    signal = np.abs(signal) #Oliver: do we need this for unit testing? It will bias the fits by forcing positive signals.
+    signal /= signal[0] # this scales noise differenly. I have now made sure signal is produced with amplitude 1 if right flags are selected. Then  SNR is fixed. Then, this sentence may be redundant
```

Comment:
Should probably be removed, at least eventually each algorithm should have their own scaling built in. For now maybe it needs to stay though?